libs/runtime/ebpf_epoch.c

```diff
@@ -379,6 +450,10 @@ ebpf_epoch_enter(_Out_ ebpf_epoch_state_t* epoch_state)
     epoch_state->epoch = cpu_entry->current_epoch;
     ebpf_list_insert_tail(&cpu_entry->epoch_state_list, &epoch_state->epoch_list_entry);
 
+    if (!cpu_entry->active) {
+        _ebpf_epoch_activate_cpu(epoch_state->cpu_id);
```

Comment:
Consider having a `_ebpf_epoch_activate_cpu_in_dispatch` variation of the function as well, to make the code more readable.